src/helpers/permit.ts

```diff
@@ -57,7 +57,7 @@ export const generatePermit2Signature = async (spender: string, amountInEth: str
 
   const base64encodedTxData = Buffer.from(JSON.stringify(txData)).toString("base64");
 
-  const result = `${permitBaseUrl}?claim=${base64encodedTxData}&chainId=0x${chainId.toString(16)}`;
+  const result = `${permitBaseUrl}?claim=${base64encodedTxData}&network=0x${chainId.toString(16)}`;
```

Comment:
I think we should use base 10 instead of base 16 to make it more human readable in the URL. 
```suggestion
  const result = `${permitBaseUrl}?claim=${base64encodedTxData}&network=${chainId}`;
```